R/EventStudy.R

```diff
@@ -167,6 +173,7 @@ EventStudy <- function(estimator, data, outcomevar, policyvar, idvar, timevar, c
     if (! is.logical(cluster)) {stop("cluster should be either TRUE or FALSE.")}
     if (FE & !cluster)         {stop("cluster=TRUE is required when FE=TRUE.")}
     if (! is.logical(anticipation_effects_normalization)) {stop("anticipation_effects_normalization should be either TRUE or FALSE.")}
+    if (! is.logical(allow_duplicate_id)) {stop("allow_duplicate_id should be either TRUE or FALSE.")}
```

Comment:
Following the excellent logic you added above, we can simplify these checks by looping over the options that are required to be Boolean!